### PR TITLE
style: minor navbar&footer styling (+ some code refactor)

### PR DESCRIPTION
**Pull request by mathiasraa**
## Proposed Changes

- Change navbar shadow
- Make loginbutton larger again :)
- Minor refactoring & style adjustments

image

## Types of Changes

- [ ] New feature
- [ ] Bug fix
- [ ] Enhancement/optimization
- [x] Refactor
- [x] Style
- [ ] Build/dependencies
- [ ] Other

## Issue(s) fixed or closed by this PR

- Closes #

## Checklist

- [ ] I have added tests to cover my changes (for features/bugs)
- [x] I am pleased with the readability of the code (if not, state where you need input)
- [x] I have tested the changes and verified that they work and do not break anything (to the best of my ability)

### CODE REVIEW

File: frontend/src/layouts/components/LoginButton.tsx [hunk 1]
```diff
@@ -10,9 +10,9 @@ const LoginButton: React.FC = () => {
   const { data } = useQuery<{ user: UserInfo | null }>(GET_USER_INFO);
 
   return (
-    <LoginRequired size="small" color="inherit" data-test-id="login">
+    <LoginRequired size="medium" color="inherit" data-test-id="login">
```
Comment (larwaa) [inline frontend/src/layouts/components/LoginButton.tsx:13]: Haha fine, I'll give you this one

File: frontend/src/layouts/footer/Footer.tsx [hunk 2]
```diff
@@ -34,17 +34,18 @@ const Watermark = styled("div")(({ theme }) => ({
...
-      <Paper sx={{ bgcolor: (theme) => theme.palette.background.neutral }}>
-        <Container sx={{ position: "relative", py: { xs: 8, md: 10 } }}>
+      <Paper sx={{ bgcolor: "background.neutral" }}>
+        <Container sx={{ position: "relative", py: { xs: 6, md: 10 } }}>
           <Grid container spacing={3} justifyContent={{ md: "space-between" }}>
             <Grid item xs={12} md={3}>
               <Stack alignItems="flex-start" spacing={3}>
                 <Logo />
-                <Typography variant="body3" sx={{ color: grey[700] }}>
+                <Typography variant="body3" sx={{ color: "text.secondary" }}>
```
Comment (larwaa) [inline frontend/src/layouts/footer/Footer.tsx:48]: The change was made as the contrast between background.neutral and text.secondary is insufficient. The change was a poor one however, as grey[700] does not work for dark mode. 

Regardless, we need something with a bit more contrast here.

**Review by larwaa — CHANGES_REQUESTED** [review]
Looks great! Very minor concerns 👇

File: frontend/src/layouts/footer/Footer.tsx (hunk 2 above)
Comment (mathiasraa) [reply frontend/src/layouts/footer/Footer.tsx:48]: Lighthouse testing doesn't complain about the contrast ratio, so it shouldn't be a UX or SEO problem.

I can agree to change it though. My main concern was the import of colors from mui, which doesn't apply the theme palette. The correct syntax in this scenario would be just "grey.700" as a string

Comment (mathiasraa) [reply frontend/src/layouts/footer/Footer.tsx]: solved in [3015ad2](https://github.com/rubberdok/indok-web/pull/679/commits/3015ad28412e9b3f7d0a9de27b534a35e44a87ea)

Comment (larwaa) [reply frontend/src/layouts/footer/Footer.tsx]: Wait, you didn't get complaints in LH? That's why I changed it 😮

Comment (larwaa) [reply frontend/src/layouts/footer/Footer.tsx]: > I can agree to change it though. My main concern was the import of colors from mui, which doesn't apply the theme palette. The correct syntax in this scenario would be just "grey.700" as a string

TIL, thanks!

**Review by larwaa — APPROVED** [review]
LGTM

**Pull request merged**